front/svelte-check-watch/public/iframe_api.js

```diff
@@ -0,0 +1,16 @@
+Copyright (c) Microsoft Corporation.
```

Comment:
This fille must delete of git. I think it's file generated when we build the fron in production mode.